monitor.py

```diff
@@ -17,20 +17,27 @@ def fetch_release_date(release):
     '''
     Returns the created date of release
     '''
-    url = constants.RELEASE_URL + release
+
     try:
+        url = constants.STABLE_RELEASE_URL + release
         response = requests.get(url, verify=False, timeout=15)
+        if response.status_code == 404:
```

Comment:
Overall LGTM
```suggestion
        if response.status_code != 200:
```
Can we have check like this? Because status_codes may change anytime so lets rely on success code only

Reply:
IMO we should check for not found only so status_code will be always 404 right. 
If we hit any other status_code(eg: 500,400) these indicates other errors. For other issues we are not supposed to proceed with checking next (dev) url.  It will be handled in this [ else part ](https://github.com/ocp-power-automation/ci-monitoring-automation/blob/b576bd443aba702c4c7ebed803fe66d782ba8b31/monitor.py#L40)